src/utilities/matrix_stats.c

```diff
@@ -142,8 +142,8 @@ hypre_MatrixStatsArrayPrint(HYPRE_Int                num_hierarchies,
    HYPRE_Int            square_count;
    HYPRE_Int            num_levels_total;
    HYPRE_Int            fine_num_rows;
-   HYPRE_Int            coarse_num_rows;
-   HYPRE_Int            total_num_rows;
+   HYPRE_BigInt         coarse_num_rows;
```

Comment:
Why is coarse_num_rows BigInt, but fine_num_rows is not?

Reply:
it has to be as well, thanks for catching!